charts/tractusx-connector/templates/deployment-controlplane.yaml

```
@@ -122,6 +122,8 @@ spec:
               value: {{ .Values.controlplane.ssi.miw.url }}
             - name: "TX_SSI_MIW_AUTHORITY_ID"
               value: {{ .Values.controlplane.ssi.miw.authorityId }}
+            - name: "TX_SSI_MIW_AUTHORITY_ISSUER"
+              value: {{ .Values.controlplane.ssi.miw.authorityIssuer }}
```

maybe we should use
```suggestion

              value: {{ .Values.controlplane.ssi.miw.authorityIssuer | quote }}
```
to avoid splitting issues. same for the other values